database_wrapper.py

```diff
@@ -634,6 +636,26 @@ def validate_and_format_entity_data(self, entity_type, data_dict: dict):
         self.validate_input_keys(data_dict, EXPECTED_KEYS_BY_ENTITY[entity_type])
         return data_dict
 
+    def add_entity(self, entity) -> bool:
+        """
+        A simplified version of insert_entity that relies on the entity performing its own formatting in the constructor call.
+        Parameters
+        ---------
+        `entity - Entity` an initialized entity object to be added to the database.
+
+        Returns
+        -------
+        `bool` whether entity was successfully added.
+        """
+        # Don't post if the entity doesn't abide by the rules of the Entity superclass
+        if not isinstance(entity, Entity):
+            return False
```

Comment:
Should this throw an error? Or is the caller expected to handle this

Reply:
We could do either. I just wanted to make sure that the only objects that get through are a subclass of Entity

Reply:
Right. If you think we are never going to want to ignore this error, I'd handle the error checking in the callee to save some time/effort in not error checking every time we call it. That's my opinion though - I'll leave it up to your judgement